WooCommerce/Classes/Extensions/UIStackView+Subviews.swift

```diff
@@ -12,4 +12,13 @@ extension UIStackView {
     func addArrangedSubviews(_ subviews: [UIView]) {
         subviews.forEach({ addArrangedSubview($0) })
     }
+
+    /// Convenience method to remove all subviews from a stack view.
+    ///
+    func removeAllSubviews() {
+        for view in self.subviews {
```

Comment:
nit: is `self` needed? can probably also use the `subviews.forEach` similar to `addArrangedSubviews` above.
```suggestion
        for view in subviews {
```

Reply:
Done in https://github.com/woocommerce/woocommerce-ios/pull/10005/commits/052309bdf7946ea7720e2bfcd16c2bc8274e676f